docs/DeveloperGuide.md

```diff
@@ -664,23 +655,27 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 ### 6.4. Non-Functional Requirements
 
-1. Should work on any _mainstream OS_ as long as it has Java `11` or above installed.
-2. Should be able to hold up to 1000 persons without a noticeable sluggishness in performance for typical usage.
-3. A user with above average typing speed for regular English text (i.e. not code, not system admin commands) should be able to accomplish most of the tasks faster using commands than using the mouse.
-4. The main window should be resizable in order to fit different screen size.
-
-*{More to be added}*
+1. Should work on any mainstream OS as long as it has Java `11` or above installed.
+2. Should be for a single user i.e. (not a multi-user product).
+3. Should be able to hold up to 1000 persons without a noticeable sluggishness in performance for typical usage.
+4. Should respond to a command within 2 seconds.
+5. Should work without requiring an installer.
+6. Should work well (i.e., should not cause any resolution-related inconveniences to the user) for standard screen resolutions 1920x1080 and higher, and screen scales 100% and 125%.
+7. The data should be stored locally and should be in a human editable text file.
+8. A user with above average typing speed for regular English text (i.e. not code, not system admin commands) should be able to accomplish most of the tasks faster using commands than using the mouse.
 
 ### 6.5. Glossary
 
-* **OS**: Operating System
-* **Mainstream OS**: Windows, Linux, Unix, OS-X
 * **Abπ**: AddressBook pi
+* **CLI**: Command-Line Interface
 * **DRY**: Don't repeat yourself
+* **GUI**: Graphical User Interface
 * **JSON**: JavaScript Object Notation
+* **MSS**: Main Success Scenario
+* **OS**: Operating System
+* **Mainstream OS**: Windows, Linux, Unix, OS-X
```

Comment:
Isn't it better to leave this in alphabetical order? You can put "Mainstream OS: Mainstream Operating Systems: Windows..." if you're worried about M coming before O.